packages/modules/src/modules/epics/ApplePaySvg.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { css, SerializedStyles } from '@emotion/react';
+import { space } from '@guardian/src-foundations';
+import { from } from '@guardian/src-foundations/mq';
+
+const applePayStyles = css`
+    display: block;
+    height: 1.1rem;
+    width: auto;
+    margin-top: ${space[2]}px;
+    /* margin-left: ${space[1]}px; */
```

Comment:
Do we need these commented out styles anymore?

Reply:
sorry not required, removed